### ISSUE TITLE
EthPortTable, PhyPortTable, PhyPortStatsTable, and PhyPortErrorTable do not support et-x/y/z interfaces

### ISSUE THREAD

**Issue by stacywsmith** [Juniper/py-junos-eznc#404]
On some models of Junos devices 100G, 40G, and even 10G Ethernet interfaces have names that begin with 'et-'.

The ethport.yml file includes this line to specify the interface-name argument to the get-interface-information RPC:
```
interface_name: '[afgx]et*'
```

This glob pattern does not match et-x/y/z interfaces.

Glob patterns are limited, so you can't get an exact match, but that's not critical as long as the glob pattern matches at least a superset of the desired interfaces. The XPath will further limit the response to just the interfaces which have the expected XML elements. Therefore, I suggest replacing with this glob pattern:
```
interface_name: '[afgxe][et]-*'
```

This will match some unnecessary (theoretical) interface names such as at-, ft-, gt-, xt, and ee-. As described before, that should be OK.

There's a similar issue in the phyport.yml file, except that pattern should not match ae- interfaces.

Currently:
```
interface_name: '[fgx]e*'
```

Change to:
```
interface_name: '[fgx][et]-*'
```

**Comment by vnitinv** [comment]
@stacywsmith Can you help me with a pull request with needed changes.

**Comment by vnitinv** [comment]
Fixed via #549
